QUICKSTART.md

```diff
@@ -48,7 +48,7 @@ In a terminal window, in the project folder:
 
 1. Type `npm install jquery@3.1.1 -S`
 2. Type `npm install @infor/sohoxi@4.6.0 -S`
```

Comment:
This line can be changed too.

`npm install ids-enterprise@4.7.0`

I actually have a complete replacement for this file in the quick start app.  https://github.com/infor-design/enterprise-ng-quickstart/blob/master/README.md

I would copy the README.md over this file.